packages/grafana-ui/src/components/Table/types.ts

```diff
@@ -33,3 +34,11 @@ export interface TableCellProps extends CellProps<any> {
 export type CellComponent = FC<TableCellProps>;
 
 export type FooterItem = Array<KeyValue<string>> | string | undefined;
+
+export type GrafanaTableColumn = Column & {
+  grafanaField: Field;
```

Comment:
Can we ust name it `field` if this doesn't conflict with properties from `Column` interface?

Reply:
@dprokop yes, but this type is really owned by react-table,  It could used in a a future release of react-table, but maybe unlikely